script/lazytox.py

```diff
@@ -0,0 +1,234 @@
+#!/usr/bin/env python
+"""
+Lazy 'tox' to quickly check if branch is up to PR standards.
+
+This is NOT a tox replacement, only a quick check during development.
+"""
+import os
+import asyncio
+import sys
+import re
+import shlex
+from collections import namedtuple
+
+try:
+    from colorlog.escape_codes import parse_colors
+except ImportError:
+    def parse_colors(_):
+        """Empty parse_colors function."""
+        return ''
+
+
+RE_ASCII = re.compile(r"\033\[[^m]*m")
+Error = namedtuple('ERROR',
+                   "file line col msg")  # pylint: disable=invalid-name
```

Comment:
I think if you'd write this the pylint error would go away (I think) *and* you'd have the actual `typename` you're using for the `namedtuple`:

```python
Error = namedtuple('Error', ['file', 'line', 'col', 'msg'])
```

(You *could* also take a look at using the [`attrs`](http://www.attrs.org/en/stable/examples.html) library that was recently added as a global Home Assistant requirement.)